subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -667,6 +667,11 @@ static void delayed_action_timeout(struct k_work *work)
 		turn_off(srv, &transition, true);
 	} else if (atomic_test_and_clear_bit(&srv->flags, FLAG_OCC_PENDING) &&
 		   (srv->state == LIGHT_CTRL_STATE_ON ||
+		    srv->state == LIGHT_CTRL_STATE_PROLONG ||
+		    (/* Fade Standby Auto */
+		     srv->state == LIGHT_CTRL_STATE_STANDBY &&
+		     !atomic_test_bit(&srv->flags, FLAG_ON) &&
+		     atomic_test_bit(&srv->flags, FLAG_TRANSITION)) ||
```

Comment:
Does this also include FADE STANDBY MANUAL, or should we also check for the "not FLAG_MANUAL".

Reply:
No it should not include that. As per Table 6.45 Fade Standby Manual state does not take any action with Occupancy event.

Reply:
This seems correct to me. Occupancy On can't turn on the light